examples/sycl/02_bmg_gemm_mixed_dtype/02_bmg_gemm_bf16_s8_bf16.cpp

```diff
@@ -576,78 +576,54 @@ int main(int argc, const char** argv)
   // Use the helpers to avoid template arg repetition
```

Comment:
In the example files you have renamed, please change the name as given in the comment at the top of the file too.